### ISSUE TITLE
False positive `SlevomatCodingStandard.Classes.ParentCallSpacing.IncorrectLinesCountBeforeControlStructure`

### ISSUE THREAD

**Issue by simPod** [slevomat/coding-standard#1345]
```php
return [
    ...parent::jsonSerialize(),
    ...[],
];
```

>  Expected 1 line before "parent", found -1.

I believe it should not trigger for `parent` calls within arrays

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/e950131822509bee520f58eef18914ce5b3e4299
